src/api/weatherApi.js

```diff
@@ -0,0 +1,145 @@
+/* eslint-env browser */
+import getHours from 'date-fns/getHours';
+import { renderToday, renderOtherDays } from '../modules/render';
+const modal = document.querySelector('.modal-window');
+
+const wetSymbls = {
+  Cloudy: 'fa-cloud',
+  Sunny: 'fa-sun',
+  Rainy: 'fa-cloud-rain',
+  Misty: 'fa-smog',
+  Extreme: 'fa-cloud-showers-heavy'
+};
+const getFormattedStatus = status => {
+  let currentStatus = status.toLowerCase();
+
+  if (currentStatus == 'rain') {
+    return 'Rainy';
+  }
+  if (currentStatus == 'clouds') {
+    return 'Cloudy';
+  }
+  if (currentStatus == 'clear') {
+    return 'Sunny';
+  }
+  if (currentStatus == 'mist') {
+    return 'Misty';
+  }
+  if (currentStatus == 'extreme') {
+    return 'Extreme';
+  }
+  return status;
+};
+
+const days = [
+  'Sunday',
+  'Monday',
+  'Tuesday',
+  'Wednesday',
+  'Thursday',
+  'Friday',
+  'Saturday'
+];
+
+const KtoC = degree => {
+  return parseInt(degree - 273.15, 10);
+};
+
+const decideData = date => {
+  let theHour = getHours(date);
+  while (theHour % 3 != 0) {
+    theHour--;
+  }
+  const first = (24 - theHour) / 3;
+  return [first + 2, first + 10, first + 18, first + 26];
+};
+
+const getWeather = name => {
+  fetch(
+    `http://api.openweathermap.org/data/2.5/weather?q=${name.toUpperCase()}&APPID=0f695179f8524df704f661f75ece5bb7`,
```

Comment:
Let's use the secure `https` link here otherwise the browser will throw a mixed content error when served from a secure server.